opentelemetry-api/src/opentelemetry/util/types.py

```diff
@@ -13,7 +13,16 @@
 # limitations under the License.
 
 
-import typing
+from typing import Union, Optional, Dict, Sequence
 
-AttributeValue = typing.Union[str, bool, float]
-Attributes = typing.Optional[typing.Dict[str, AttributeValue]]
+AttributeValue = Union[
+    str,
+    bool,
+    int,
```

Comment:
Note, the `int` is also new.

Reply:
Updated the title to reflect the change.